src/ProjectDataProvider.ts

```diff
@@ -319,23 +319,35 @@ export module ProjectDataProvider {
       let vscodeRootpath = item.replace('go.mod', '');
       let targetDir = paths.join(vscodeRootpath, 'target');
       const goGraphFilePath = paths.join(targetDir, 'golist.json');
+      const goManifestPath = paths.join(paths.join(targetDir, 'bin'), 'gomanifest');
 
       if (!fs.existsSync(targetDir)) {
         fs.mkdirSync(targetDir);
       }
 
       const cmd: string = [
+        `export`,
+        `CRDA_GOPATH=$GOPATH`,
+        `&&`,
+        `export`,
+        `GOPATH=${targetDir}`,
+        `&&`,
```

Comment:
This is incorrect and will not work in Windows. If you want to set an env variable, use `env` option.

e.g.

```
   exec(
        cmd,
        { maxBuffer: 1024 * 1200, env: Object.assign(process.env, {"GOPATH": `"${targetDir}"`}) }
....
```

Reply:
@arajkumar  Now getting temporary path from OS and setting that as GOPATH to get `gomanifest` module. Also GOPATH is set in ENV and passed to exec command. PTAL